frontend/projects/upgrade/src/app/features/dashboard/home/components/experiment-participants/experiment-participants.component.ts

```diff
@@ -205,10 +202,12 @@ export class ExperimentParticipantsComponent implements OnInit {
 
     if (includedMembersFiltered.length === 0) {
       this.members1.clear();
+      this.updateView1();
```

Comment:
what happens if this isn't included?

Reply:
So the problem with the previous code without the `updateView1()` is that if we create an empty row and move to the next step, it clears the `members1` table but it is not applied in the UI so even if you fill the row the next time, the row is not actually being added. Calling `updateView1()` fixes this by removing an empty/unfilled row in the UI.

https://user-images.githubusercontent.com/90279765/183509866-9d97557c-959f-4fda-886d-4fe50f1d816d.mov

Reply:
👍